crates/next-api/src/nft_json.rs

```diff
@@ -123,6 +208,44 @@ impl Asset for NftJsonAsset {
             .copied()
             .chain(std::iter::once(chunk))
             .collect();
+
+        let exclude_globs: Option<Vec<Glob>> = if let Some(route) = &this.page_name {
+            let project_path = this.project.project_path().await?;
+
+            if let Some(excludes_config) = output_file_tracing_excludes {
+                let mut combined_excludes: BTreeSet<RcStr> = BTreeSet::new();
+
+                if let Some(excludes_obj) = excludes_config.as_object() {
+                    for (glob_pattern, exclude_patterns) in excludes_obj {
+                        // Check if the route matches the glob pattern
+                        if let Ok(glob) = Glob::parse(glob_pattern)
+                            && glob.matches(route)
+                            && let Some(patterns) = exclude_patterns.as_array()
+                        {
+                            for pattern in patterns {
+                                if let Some(pattern_str) = pattern.as_str() {
+                                    combined_excludes
+                                        .insert(format!("{project_path}/{pattern_str}").into());
+                                }
+                            }
+                        }
+                    }
+                }
+
+                Some(
+                    combined_excludes
+                        .iter()
+                        .map(|pattern| Glob::parse(pattern.as_str()))
+                        .collect::<Result<Vec<Glob>, _>>()?,
+                )
```

Comment:
it is more efficient to produce a single glob, especially since they all have the same prefix. 

```suggestion
                            for pattern in patterns {
                                if let Some(pattern_str) = pattern.as_str() {
                                    combined_excludes.insert(pattern_str.into());
                                }
                            }
                        }
                    }
                }

                Some(
                    Globl::parse(format!("{project_path}/{{{}}}", 
                    combined_excludes
                        .iter().collect::<Vec<_>>().join(","))?
                )
```

Reply:
see also https://github.com/vercel/next.js/blob/0e73aeea6a59f0f468f07c130882a8c0d1490c05/turbopack/crates/turbo-tasks-fs/src/glob.rs#L101